ndk/src/events/nip13/Miner.ts

```diff
@@ -0,0 +1,72 @@
+import type { UnsignedEvent } from "nostr-tools";
```

Comment:
Curious why you've separated this `Miner.ts` file out into it's own thing instead of including it with the `nip13.ts` file below?

Reply:
This is kindof a holdover from pulling code from ONOSENDAI. I can merge the code in Miner into `nip13.ts`. Good call.

Reply:
@erskingardner Oh, pow.worker.ts and nip13.ts both require functions from Miner.ts. If I moved the functions from Miner.ts into nip13.ts, then we'd have nip13.ts and pow.worker.ts requiring files from each other creating a circular dependency which might not be great. To avoid this, we should keep the functions separate in Miner.ts, but we can rename it to something better perhaps.